api/types.proto

```diff
@@ -339,6 +341,26 @@ message RestartPolicy {
 	// The time window is unbounded if this is 0.
 	// Note: can't use stdduration because this field needs to be nullable.
 	google.protobuf.Duration window = 4;
+
+	// Defines the backoff policy used when restarting this task.
+	// The calculated backoff will be a random value from 0 to
+	// min(Backoff.Max, Delay + Backoff.Factor * (2 ** failures))
+	BackoffPolicy backoff = 5;
+}
+
+message BackoffPolicy {
```

Comment:
This is not so much a policy, as it is a configuration.

Reply:
We were following the naming convention of RestartPolicy which this is a part of.